pypsa/statistics.py

```diff
@@ -119,6 +129,11 @@ def get_ports(n, c):
     return [col[3:] for col in n.df(c) if col.startswith("bus")]
 
 
+def get_stacked_ports(n, c):
+    ports = get_ports(n, c)
+    return pd.concat([n.df(c)[f"bus{p}"][lambda ds: ds != ""] for p in ports])
+
+
```

Comment:
I think we can remove this, right? Was an intermediate alternative route from me.

Reply:
will do so.